front/pages/api/w/[wId]/assistant/conversations/[cId]/content_fragment/index.ts

```diff
@@ -0,0 +1,165 @@
+import { isLeft } from "fp-ts/lib/Either";
+import * as t from "io-ts";
+import * as reporter from "io-ts-reporters";
+import { NextApiRequest, NextApiResponse } from "next";
+
+import {
+  getConversation,
+  postNewContentFragment,
+} from "@app/lib/api/assistant/conversation";
+import { Authenticator, getSession } from "@app/lib/auth";
+import { ReturnedAPIErrorType } from "@app/lib/error";
+import { apiError, withLogging } from "@app/logger/withlogging";
+import { ContentFragmentType } from "@app/types/assistant/conversation";
+
+export const PostContentFragmentRequestBodySchema = t.type({
```

Comment:
why are we duplicating this ?

Reply:
@fontanierh between api/v1 and api/ these are not exactly the same.
In v1, we have properties that we derive from the user (`PostContentFragmentRequestBodySchema.context.email` for example) while in the api/v1 case, we don't have a user object and we have to rely on the user input.

Reply:
Got it 👍